### PR TITLE
Replace some `JSON.parse` calls

### PR DESCRIPTION
**Pull request by eventualbuddha**
image

### CODE REVIEW

File: apps/bas/src/App.tsx [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
   VoterCardData,
 } from '@votingworks/types'
+import { Card } from '@votingworks/utils'
 
-import fetchJSON from './utils/fetchJSON'
-
-import { CardAPI, EventTargetFunction } from './config/types'
+import { EventTargetFunction } from './config/types'
 
 import useStateAndLocalStorage from './hooks/useStateWithLocalStorage'
```
Comment (carolinemodic) [inline apps/bas/src/App.tsx:14]: I know this isn't really related to this PR but its a little confusing and seems unneccessary that this is renamed from useStateWithLocalStorage to useStateAndLocalStorage

File: apps/bas/src/App.tsx [hunk 2]
```diff
@@ -141,6 +149,7 @@ const App: React.FC = () => {
         break
     }
 
+    console.log({ isWritableCard })
```
Comment (carolinemodic) [inline apps/bas/src/App.tsx:152]: remove

**Review by carolinemodic — APPROVED** [review]

**Comment by eventualbuddha** [comment]
I don't understand why [this build failure](https://app.circleci.com/pipelines/github/votingworks/vxsuite/1734/workflows/57f9a202-fe68-4988-844f-1ccafdebdd52/jobs/21248?invite=true#step-106-30) is happening:

```
/home/circleci/project/libs/utils/src/index.ts 8:9
Module parse failed: Unexpected token (8:9)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
| export * from './deferred'
| export * from './find'
> export * as format from './format'
| export * from './fetchJSON'
| export * from './filenames'
```

**Comment by eventualbuddha** [comment]
> I don't understand why [this build failure](https://app.circleci.com/pipelines/github/votingworks/vxsuite/1734/workflows/57f9a202-fe68-4988-844f-1ccafdebdd52/jobs/21248?invite=true#step-106-30) is happening:
> 
> ```
> /home/circleci/project/libs/utils/src/index.ts 8:9
> Module parse failed: Unexpected token (8:9)
> You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
> | export * from './deferred'
> | export * from './find'
> > export * as format from './format'
> | export * from './fetchJSON'
> | export * from './filenames'
> ```

I figured it out. We had an import (helpfully supplied by VS Code 😡) with the path `@votingworks/utils/src`. This is not the build directory, but the source directory. So webpack tries to load `src/index.ts` in `@votingworks/utils` and eventually hits something that isn't JS as it understands it. "Fixing" the above error manually by using `import *` and re-exporting just makes it fail on an `interface` declaration. The correct fix is to change the path to simply `@votingworks/utils` and let it find `build/index.js`.

**Pull request merged**